src/test/java/vehicle/PutVehicleHgvFieldRestrictionsAdrUpdate.java

```diff
@@ -27,7 +27,7 @@
 import static io.restassured.RestAssured.given;
 import static util.WriterReader.saveUtils;
 
-@Ignore
+@Ignore("vtm test ignored with no description")
```

Comment:
Should this say why the tests are being ignored?

Reply:
sure, they seem to be out of date so i've labelled it as that now, initially they were just labelled with ignore without a description